include/roboteam_utils/normalize.h

```diff
@@ -14,16 +14,98 @@
 
 namespace roboteam_utils {
 
-  void rotate(proto::WorldBall *ball) ;
-  void rotate(proto::WorldRobot *robot);
-  void rotate(proto::World *world);
-  void rotate(proto::SSL_Referee *refereeData);
-  void rotate(proto::SSL_FieldCicularArc *arc);
-  void toMeters(proto::SSL_FieldCicularArc *arc);
-  void rotate(proto::SSL_FieldLineSegment *line);
-  void toMeters(proto::SSL_FieldLineSegment * line);
-  void rotate(proto::SSL_GeometryFieldSize * field);
-//  void toMeters(proto::SSL_GeometryFieldSize * field);
-  void rotate(proto::RobotCommand * command);
+    /**
+     * @brief Multiplies all the members of a Ball by -1
+     * 
+     * Pointer is asserted in debug mode
```

Comment:
What is ' debug mode' here ? Do you mean when the application is compiled as release version, assert() is not actually added to the binary/assembly?

Reply:
Correct.

https://en.cppreference.com/w/cpp/error/assert

```
The definition of the macro assert depends on another macro, NDEBUG, which is not defined by the standard library.

If NDEBUG is defined as a macro name at the point in the source code where <cassert> is included, then assert does nothing.

```

Reply:
Is this 100% done by CMake? This seems interesting, we would probably use a lot more asserts if we can guarantee that it only crashes our programs in debug and not in release

Reply:
https://stackoverflow.com/questions/34302265/does-cmake-build-type-release-imply-dndebug
Yup.